test/__fixtures/utils/happn-test-helper.js

```diff
@@ -0,0 +1,54 @@
+const happn = require('../../../..');
```

Comment:
I don't think I am actually using this in any of the new tests, will remove and test